### PR TITLE
Add combo

### PR DESCRIPTION
**Pull request by apcamargo**
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by apcamargo** [comment]
Hi @yzhao062!

Can you confirm in here that you have agreed to be a maintainer of this recipe?

**Comment by yzhao062** [comment]
> Hi @yzhao062!
> 
> Can you confirm in here that you have agreed to be a maintainer of this recipe?

Yes please :)

File: recipes/combo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - joblib
+    - matplotlib
```
Comment (scopatz) [inline recipes/combo/meta.yaml]: This should be `matplotlib-base`

Comment (apcamargo) [reply recipes/combo/meta.yaml]: Done!

File: recipes/combo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - combo
+    - combo.models
+    - combo.utils
+
+about:
+  home: "https://github.com/yzhao062/combo"
+  license: BSD
```
Comment (npavlovikj) [inline recipes/combo/meta.yaml]: This is a minor comment, but the license should be `BSD-2-Clause` instead.

File: recipes/combo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/combo/meta.yaml]:
```suggestion
    - python
    - setuptools >=38.6.0
```

File: recipes/combo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - joblib
+    - matplotlib-base
+    - numba >=0.35
+    - numpy >=1.13
+    - python
+    - scikit-learn >=0.20
+    - scipy
```
Comment (marcelotrevisani) [inline recipes/combo/meta.yaml]: I might be wrong, but it seems this project also depends on ``pyod``, am I right?

File: recipes/combo/meta.yaml (hunk 3 above)
Comment (scopatz) [reply recipes/combo/meta.yaml]: people should not add setuptools if they have pip

File: recipes/combo/meta.yaml (hunk 4 above)
Comment (scopatz) [reply recipes/combo/meta.yaml]: I think there was a circular dependency and pyod was optional

File: recipes/combo/meta.yaml (hunk 2 above)
Comment (apcamargo) [reply recipes/combo/meta.yaml]: Done!

File: recipes/combo/meta.yaml (hunk 4 above)
Comment (apcamargo) [reply recipes/combo/meta.yaml]: It's optional: https://github.com/conda-forge/staged-recipes/pull/10964#issuecomment-594653410

**Pull request closed without merging**